### PR TITLE
docs: fix formatting

### PR DESCRIPTION
**Pull request by chaoran-chen**
Using prettier to fix formatting of the docs

### CODE REVIEW

File: docs/src/content/docs/for-administrators/my-first-loculus.md [hunk 1]
```diff
@@ -134,50 +126,48 @@ Loculus ships with some default organisms, but you probably want to overwrite th
 Let's edit the `custom_values.yaml` file to the following:
 
 ```yaml
```
Comment (theosanderson) [inline docs/src/content/docs/for-administrators/my-first-loculus.md:128]: IMO it probably makes sense to keep this consistent with our yaml files which seem to have tabs as two spaces

Comment (corneliusroemer) [reply docs/src/content/docs/for-administrators/my-first-loculus.md:128]: I made it 2 spaces to save some horizontal space as the github actions yamls can get quite nested - maybe prettier doesn't support that type of configuration in md.

**Review by corneliusroemer — COMMENTED** [review]
We should add prettier formatting to CI if we want this to be maintained as opposed to being a one-off :)

File: docs/src/content/docs/for-administrators/my-first-loculus.md (hunk 1 above)
Comment (chaoran-chen) [reply docs/src/content/docs/for-administrators/my-first-loculus.md:129]: ok, I added an ignore to avoid formatting this.

**Comment by chaoran-chen** [comment]
@corneliusroemer, yes, I agree! I just moved #2419 to the Prio 1 milestone.

**Review by theosanderson — APPROVED** [review]

**Pull request merged**